### ISSUE TITLE
D clone didn't appear

### ISSUE THREAD

**Issue by torrin5000** [haxifix/PlugY#7]
Was in DoH level 3 when D walked the earth. Since all the superuniques on that level were spawned, D wouldn't appear in DoH. When the message came up, I immediately TPed to town and headed to the Chaos Sanctuary to have him appear there. However, he didn't appear as any of the three seal superuniques. Then headed to Frigid WP and he didn't appear there either. Bug?

**Comment by Warren1001** [comment]
where were you standing in durance 3 when he walked? also, if hephasto is close enough to the river wp, he could spawn there first.

**Comment by torrin5000** [comment]
Yes, standing in durance level 3. Didn't check for hephasto, so maybe that
was it.

**Comment by Warren1001** [comment]
yes, but where in the durance? monsters spawn within ~4? screens. if you were just inside the entrance, one of the travincal members could have become clone if you then moved close enough. various possibilities

**Comment by torrin5000** [comment]
I checked the entire durance level 3 when I saw the d clone message, and
none had changed into the d clone.

**Comment by Warren1001** [comment]
could ahve been hephasto then. ive also heard reports of clone being weird with chaos seal popping exploit. see if you can get him to spawn randomly elsewhere before trying to seal trick to ensure theres no bug with simple spawns
/e to clarify - this comes from original plugy, and would not be caused by 12.00
